server/pendinggame.js

```diff
@@ -100,34 +100,19 @@ class PendingGame {
         };
     }
 
-    newGame(id, user, password, callback) {
+    newGame(id, user, password) {
         if(password) {
-            bcrypt.hash(password, 10, (err, hash) => {
-                if(err) {
-                    logger.info(err);
-
-                    callback(err);
-
-                    return;
-                }
-
-                this.password = hash;
-                this.addPlayer(id, user);
-
-                callback();
-            });
-        } else {
-            this.addPlayer(id, user);
-
-            callback();
+            this.password = crypto.createHash('md5').update(password).digest('hex');
```

Comment:
Have to admit, even though this is ephemeral (only lasts as long as the individual game), it makes me nervous (perhaps irrationally). What spurred dropping bcrypt here?

Reply:
To get rid of it being async.  It fixes the 'can join multiple games at once' bug.  It could as well store the plain text, it's really just a shared key between two people, and not meant to be protecting anything secure